sdk-extension/opentelemetry-sdk-extension-aws/README.rst

```diff
@@ -65,6 +65,37 @@ Or by setting this propagator in your instrumented application:
 
     set_global_textmap(AwsXRayFormat())
 
+Usage (AWS Resource Detectors)
+------------------------------
+
+Use the provided `Resource Detectors` to automatically populate attributes under the `resource`
+namespace of each generated span.
+
+For example, if tracing with OpenTelemetry on an AWS EC2 instance, you can automatically
+populate `resource` attributes by creating a `TraceProvider` using the `AwsEc2ResourceDetector`:
+
+.. code-block:: python
+
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.extension.aws.resource.ec2 import (
+        AwsEc2ResourceDetector,
+    )
+    from opentelemetry.sdk.resources import OTELResourceDetector
+
+    trace.set_tracer_provider(
```

Comment:
Missing import  for `trace`?

Reply:
Thank you! Although actually I think I thought about removing even `from opentelemetry.sdk.trace import TracerProvider` because I already imported them in the `IdsGenerator` example above... but I think it's better to let this doc section stand alone and it's just 2 lines.

Added it in!